packages/fleather/lib/util.dart

```diff
@@ -53,3 +53,12 @@ bool isDataOnlyNewLines(Object data) {
   if (data is! String || data.isEmpty) return false;
   return RegExp('^(\n)+\$').hasMatch(data);
 }
+
+int _floatToInt8(double x) => (x * 255.0).round() & 0xff;
```

Comment:
Don't we want to wrap these function in a `value32Bits` method in an extension on `Color`?

Reply:
That makes sense and reads better indeed! @amantoux